twitter-workitem/src/main/java/org/jbpm/process/workitem/twitter/UpdateStatusWorkitemHandler.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.process.workitem.twitter;
+
+import java.io.File;
+import java.io.FileOutputStream;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import org.jbpm.document.Document;
+import org.jbpm.process.workitem.core.AbstractLogOrThrowWorkItemHandler;
+import org.jbpm.process.workitem.core.util.Wid;
+import org.jbpm.process.workitem.core.util.WidMavenDepends;
+import org.jbpm.process.workitem.core.util.WidParameter;
+
+import org.kie.api.runtime.process.WorkItem;
+import org.kie.api.runtime.process.WorkItemManager;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import twitter4j.StatusUpdate;
+import twitter4j.Twitter;
+
+@Wid(widfile = "TwitterUpdateStatus.wid", name = "TwitterUpdateStatus",
+        displayName = "TwitterUpdateStatus",
+        defaultHandler = "mvel: new org.jbpm.process.workitem.twitter.UpdateStatusWorkitemHandler()",
+        parameters = {
+                @WidParameter(name = "Message"),
+                @WidParameter(name = "Media")
+        },
+        mavenDepends = {
+                @WidMavenDepends(group = "org.twitter4j", artifact = "twitter4j-core", version = "4.0.6")
+        })
+public class UpdateStatusWorkitemHandler extends AbstractLogOrThrowWorkItemHandler {
```

Comment:
isn't the name bit confusing? UpdateStatus while it actually sends messages? or maybe I just don't understand twitter

Reply:
I think this is fine - code calls twitter.updateStatus. I think you can send direct messages (the "@" someone) and I'll add a workitem for that then as separate thing